File: Oops Design/java/Overriding.java/Overriding.java
class Bank{

	public double getRateOfinterset(){
		return 0;
	}
}

class SBI extends Bank{

	public double getRateOfinterset(){
		return 7.2;
	}
}

class AXIS extends Bank{
	public double getRateOfinterset(){
		return 7.5;
	}
}

public class Main{
	public static void main(String[] args) {
		Bank obj1=new SBI();
		Bank obj2=new AXIS();
		System.out.println(obj1.getRateOfinterset());
		System.out.println(obj2.getRateOfinterset());
	}
}

File: Oops Design/c++/overriding.cpp
class Bank{
public:
	virtual double getRateOfInterest(){
		return 0;
	} 
};

class SBI: public Bank{
public:
	double getRateOfInterest(){
		return 7.2;
	}
};

class AXIX: public Bank{
public:
	double getRateOfInterest(){
		return 7.5;
	}
};

int main(){
	Bank *obj1=new SBI();
	Bank *obj2=new AXIX();
	cout<<obj1->getRateOfInterest()<<endl;
	cout<<obj2->getRateOfInterest()<<endl;
}
